Packages/Core/Editor/Editors/AtomEventEditor.cs

```diff
@@ -1,6 +1,10 @@
-#if UNITY_2019_1_OR_NEWER
+#if UNITY_2018_4_OR_NEWER
 using UnityEngine;
+#if UNITY_2018_4_OR_NEWER
```

Comment:
The UNITY_2019_1_OR_NEWER condition should be before UNITY_2018_4_OR_NEWER, the second if-statement will never be used when running Unity 2019 since the UNITY_2018_4_OR_NEWER flag will also be true in this case.